src/DataAccess/DoctrineEntities/Subscription.php

```diff
@@ -0,0 +1,198 @@
+<?php
+
+declare( strict_types = 1 );
+
+namespace WMDE\Fundraising\SubscriptionContext\DataAccess\DoctrineEntities;
+
+use DateTime;
+use Doctrine\ORM\Mapping as ORM;
+use Gedmo\Mapping\Annotation as Gedmo;
+
+/**
+ * @since 2.0
+ *
+ * @ORM\Table( name="subscription" )
+ * @ORM\Entity
+ */
+class Subscription {
+
+	/**
+	 * @var string
+	 *
+	 * @ORM\Column(name="email", type="string", length=250, options={"default":""}, nullable=false)
+	 */
+	private $email = '';
+
+	/**
+	 * @var DateTime|null
+	 *
+	 * @ORM\Column(name="export", type="datetime", nullable=true)
+	 */
+	private $export;
+
+	/**
+	 * @var DateTime|null
+	 *
+	 * @ORM\Column(name="backup", type="datetime", nullable=true)
+	 */
+	private $backup;
+
+	/**
+	 * @var integer
+	 *
+	 * @ORM\Column(name="status", type="smallint", options={"default":0}, nullable=true)
+	 */
+	private $status = 0;
+
+	/**
+	 * @var string
+	 *
+	 * @ORM\Column(name="confirmationCode", type="string", length=32, nullable=true)
+	 */
+	private $confirmationCode;
+
+	/**
+	 * @var integer
+	 *
+	 * @ORM\Column(name="id", type="integer")
+	 * @ORM\Id
+	 * @ORM\GeneratedValue(strategy="IDENTITY")
+	 */
+	private $id;
+
+	/**
+	 * @var string
+	 *
+	 * @ORM\Column(name="tracking", type="string", length=50, nullable=true)
+	 */
+	private $tracking;
+
+	/**
+	 * @var string
+	 *
+	 * @ORM\Column(name="source", type="string", length=50, nullable=true)
+	 */
+	private $source;
+
+	/**
+	 * @var DateTime
+	 * @Gedmo\Timestampable(on="create")
+	 * @ORM\Column(type="datetime")
+	 */
+	private $createdAt;
+
+	private const STATUS_NEW = 0;
+	private const STATUS_CONFIRMED = 1;
+	private const STATUS_MODERATION = 2;
```

Comment:
We can drop this status and all moderation-related code. Moderation was only needed for questionable addresses ("Dr. Fakey McFakeface")

Reply:
Is `STATUS_CONFIRMED` used for email validation?

Reply:
yes, see https://github.com/wmde/fundraising-subscriptions/blob/master/src/UseCases/ConfirmSubscription/ConfirmSubscriptionUseCase.php